tedana/resources/decision_trees/kundu.json

```diff
@@ -0,0 +1,267 @@
+{
+    "tree_id": "kundu_MEICA27_decision_tree",
+    "info": "Following the full decision tree designed by Prantik Kundu",
+    "report": "This is based on the minimal criteria of the original MEICA decision tree without the more agressive noise removal steps",
+    "refs": "Kundu 2013",
+    "necessary_metrics": [
```

Comment:
I think just `metrics` would be cleaner here.

Reply:
Thoughts @handwerkerd ?

Reply:
There are two terms used in the code `necessary_metrics` and `used_metrics`. `necessary_metrics are declared up-front. With this input, we can take a tree, make a list of all necessary metrics and calculate only those metrics. `used_metrics` is added to has metrics are used by the decision tree. At the end, there is a check to make sure no used metrics were undeclared in `necessary_metrics`. 

The other reason I used this terminology is, if we eventually do set up the code to calculate metrics based on a decision tree, then we can have `necessary_metrics` that are used in the tree and something like `additional_metrics` which should be calculated, but not used.

I'm not wed to this exact terminology, and am open to ideas for better descriptive terms, but I think `metrics` alone is insufficient.